api/MediaKeySystemConfiguration.json

```diff
@@ -112,10 +112,10 @@
               "version_added": "≤79"
             },
             "firefox": {
-              "version_added": null
+              "version_added": "50"
```

Comment:
This (`distinctiveIdentifier`) came from https://github.com/mozilla/gecko-dev/commit/e091dd26b182e6f8beaa854f98ed98f3dd9955d1

Reply:
Confirmed by:
https://hg.mozilla.org/releases/mozilla-release/file/FIREFOX_49_0_RELEASE/dom/webidl/MediaKeySystemAccess.webidl (not there)
https://hg.mozilla.org/releases/mozilla-release/file/FIREFOX_50_0_RELEASE/dom/webidl/MediaKeySystemAccess.webidl#l29